### PR TITLE
add bind host option to alerta daemon

### PR DESCRIPTION
**Pull request by ilcic**
Add bind host option to alerta daemon instead of always binding 0.0.0.0

### CODE REVIEW

File: alerta/app/shell.py [hunk 1]
```diff
@@ -30,4 +37,4 @@ def main():
 
     LOG.info('Starting alerta version %s ...', __version__)
     LOG.info('Using MongoDB version %s ...', db.get_version())
-    app.run(host='0.0.0.0', port=args.port, debug=args.debug, threaded=True)
+    app.run(host='args.host', port=args.port, debug=args.debug, threaded=True)
```
Comment (satterly) [inline alerta/app/shell.py]: Are those single quotes meant to be there?

Comment (ilcic) [inline alerta/app/shell.py]: sorry that's an typo

**Comment by satterly** [comment]
Thanks. Happy to merge this. Curious as to what your particular use case is though.

**Comment by ilcic** [comment]
We don't want to bind the ip on virtual interface create by HAProxy.

**Comment by satterly** [comment]
Makes sense. Alerta server version 4.8.8 is now available on Pypi with this feature. Thanks.

**Pull request merged**